### PR TITLE
Package the installer as a rpm. 

### LINKED ISSUE #12: User Story: Convert the installation upstream script to an RPM
This story will convert the installation scripts to an RPM.  This will allow the upstream and downstream installer to function the same way.  This also unifies the upstream and downstream documentation for installation.

### LINKED ISSUE #80: Create a man page for the installation rpm

### PR DESCRIPTION
**Pull request by myersCody**
How to test
```
make  setup-local-online 
make test-all
sudo su
cd /quipucords_installer
make setup
```
Get link to rpm from here:  https://github.com/quipucords/quipucords-installer/releases/tag/0.0.0
```
yum install -y <rpm_url>
man quipucords-installer
quipucords-installer -h
quipucords-installer
```
closes #80  - Create a Manpage
closes #12   - Package upstream as an rpm

### CODE REVIEW

File: install/quipucords-installer.1 [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.\" Contact quipucords@redhat.com to correct errors or typos.
+.TH man 8 "05 Aug 2019" "1.2.0"
+.SH NAME
+quipucords-installer
+.SH SYNOPSIS
+qpc-installer install
+.SH DESCRIPTION
+Installer for quipucords
+.SH USAGE
+quipucords-installer install
```
Comment (abaiken) [inline install/quipucords-installer.1]: should we remove install here?

File: install/quipucords-installer.1 [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.\" Manpage for quipucords-installer
+.\" Contact quipucords@redhat.com to correct errors or typos.
+.TH man 8 "05 Aug 2019" "1.2.0"
+.SH NAME
+quipucords-installer
+.SH SYNOPSIS
+qpc-installer install
```
Comment (abaiken) [inline install/quipucords-installer.1]: should we remove install here?

File: install/quipucords-installer.1 [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+         -e open_port=false
+    * Quipucords server port (defaults to 9443):
+         -e server_port=8443
+    * Postgres db user (defaults to 'postgres'):
+         -e dbms_user=postgres
+    * Postgres db password (defaults to 'password')
+         -e dbms_password=password
+.SH BUGS
+No known bugs.
+.SH AUTHOR
```
Comment (abaiken) [inline install/quipucords-installer.1]: should this be an authors section that has everyone who has contributed to the installer? Or is it like the author of the man page?

File: quipucords-installer.spec [hunk 4]
```diff
@@ -0,0 +1,54 @@
+%global stream_name quipucords
+%global ver 0.1.2
```
Comment (abaiken) [inline quipucords-installer.spec]: is this the correct version?

File: install/quipucords-installer.1 [hunk 5]
```diff
@@ -0,0 +1,38 @@
+.\" Manpage for quipucords-installer
+.\" Contact quipucords@redhat.com to correct errors or typos.
+.TH man 8 "05 Aug 2019" "1.2.0"
```
Comment (abaiken) [inline install/quipucords-installer.1]: "1.2.0" differs from the 0.1.2 below - not sure if it matters

**Comment by abaiken** [comment]
where are we documenting the yum commands for installing the rpm? 
```
yum install -y https://github.com/quipucords/quipucords-installer/releases/download/0.0.0/test-quipucords-installer-0.1.2-1.git.62.4e5deeb.el6.noarch.rpm
-- or ---
yum install -y https://github.com/quipucords/quipucords-installer/releases/download/0.0.0/test-quipucords-installer-0.1.2-1.git.62.4e5deeb.el7.noarch.rpm
```
Like should we add make commands for those or document them somewhere other than the description of the pr?

File: install/quipucords-installer.1 (hunk 3 above)
Comment (myersCody) [reply install/quipucords-installer.1]: All of the examples I saw just had one. We could do `Quipucords Team`

File: quipucords-installer.spec (hunk 4 above)
Comment (myersCody) [reply quipucords-installer.spec]: Yeah, I messed up the man page version sorry for the confusion.

File: docs/man.rst [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+  ``
+  quipucords-installer -e server_version=0.9.0  -e cli_vesion=0.9.0
+  ``
+
+Only CLI/Server Install
+-----------------------
+The basic installation will install the quipucords server and the qpc cli at the same time. However, you may choose to install the cli and server on seperate machines.
+
+Server only
```
Comment (abaiken) [inline docs/man.rst]: Change this heading to either `Installing the Server without the CLI` or `Installing without the CLI`

And before the command you could put an excerpt that is like: 
```
The following example command will install the Quipucords server but will not install the CLI. 
```

File: docs/man.rst [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+~~~~~~~~~~~
+
+  ``
+  quipucords-installer -e install_cli=false
+  ``
+
+Cli only
```
Comment (abaiken) [inline docs/man.rst]: Change this heading to either `Installing without the Server` or `Installing the CLI without the Server`

And before the command you could put an excerpt that is like: 
```
The following example command will install the QPC CLI but will not install the Quipucords Server. 
```

File: docs/man.rst [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+  Specifies the database password for postgres. Defaults to password.
+
+``-e server_username=cody``
+
+  Quipucords server username. Defaults to admin.
+
+``-e server_password=c0dy``
+
+  Quipucords server password. Defaults to qpcpassw0rd.
```
[the remark this answers is not in the document]
Comment (myersCody) [reply docs/man.rst]: Should we do this for all of the default values?

[the remark this answers is not in the document]
Comment (abaiken) [reply docs/man.rst]: I think so. Most of the defaults in our QPC manpage are enclosed.

[the remark this answers is not in the document]
Comment (myersCody) [reply docs/man.rst]: Sweet! Will do, thanks ash

File: docs/man.rst [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+
+``-e server_username=cody``
+
+  Quipucords server username. Defaults to ``admin``.
+
+``-e server_password=c0dy``
+
+  Quipucords server password. Defaults to ``qpcpassw0rd``.
+
+``server_user_email=cody@hotmail.com``
```
Comment (abaiken) [inline docs/man.rst]: this is missing the `-e`

File: docs/man.rst [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+
+Additionally you can pass a flag to obtain the command usage.
+
+**quipucords-installer (-h | --help)**
+
+Optional flags can be set to control extra-variables passed to the ansible playbook.
+
+**quipucords-installer (-e | --extra-vars) key=** *value*
+
+**Keys**
```
Comment (abaiken) [inline docs/man.rst]: It seems a little strange to call these keys - I see what you are trying to do with the key=value above, but maybe we should do option=value or maybe @kholdaway has a better idea.

File: README.md [hunk 11]
```diff
@@ -155,8 +155,14 @@ make install-offline server_version=<server_version> cli_version=<cli_version>
...
  - Optionally run any secret post install scripts you included in `installer_config.tar.gz`
- - You can replace `make install` with other commands or `cd install;./install.sh -e other_flags`
+ - You can replace `make install` with other commands or `cd install;./quipucords-installer -e other_flags`
 
+ ## Creating the man page
+ After installing [pandoc](https://pandoc.org/installing.html) locally, run the following command:
+
+```
+ make manpage
+```
```
Comment (abaiken) [inline README.md]: maybe say how to view it after running `make manpage`

Comment (myersCody) [reply README.md]: done :)

File: docs/man.rst [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+
+``-e install_cli=false``
+
+  Contains a true or false value. Defaults to ``true``. Supply false to skip the installation of the command line tool.
+
+``-e install_offline=true``
+
+  Contains a true or false value. Defaults to ``false``. Supply true to start an offline installation.
+
+``-e server_version=0.0.46``
```
Comment (kholdaway) [inline docs/man.rst]: Could we bump this to 0.9.0 for the sample?  Small request, but we want people using new stuff.  Right now it is the latest, but in a week it won't be.

File: docs/man.rst [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+
+  Contains the semantic versioning format (x.y.z) of the quipucords server you want installed. Required if ``install_offline=true`` and ``install_server=true``.
+
+``-e cli_vesion=0.0.46``
```
Comment (kholdaway) [inline docs/man.rst]: same.  0.9.0

File: docs/man.rst [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
+Installing the CLI without the Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The following example command will install the QPC CLI but will not install the Quipucords Server.
+
+``quipucords-installer -e install server=false``
+
+Authors
+-------
+
+The Quipucords Installer was originally written by Chris Hambridge <chambrid@redhat.com>, Kevan Holdaway <kholdawa@redhat.com>, Ashley Aiken <aaiken@redhat.com>, Cody Myers <cmyers@redhat.com>, Cecilia Carter <cecarter@redhat.com>, and Dostonbek Toirov <dtoirov@redhat.com>.
```
Comment (kholdaway) [inline docs/man.rst]: Cecilia Carter <cecarter@redhat.com> never worked on the installer.  Let's pull her out.

**Pull request merged**